main.nf

```diff
@@ -157,13 +154,19 @@ output {
         path 'multiqc'
     }
     'rsem' {
-        path 'rsem'
+        path { meta, index -> { file -> "${meta.species}/${meta.source}/${meta.id}/Sequence/RSEMIndex/" } }
     }
     'salmon' {
-        path 'salmon'
+        path { meta, index -> { file -> "${meta.species}/${meta.source}/${meta.id}/Sequence/SalmonIndex/" } }
+    }
+    'splice_sites' {
+        path { meta, txt -> { file -> "${meta.species}/${meta.source}/${meta.id}/Sequence/SpliceSites/${file}" } }
```

Comment:
Why not in a config file ?

Reply:
This doesn't feel nf-core like 🤔

Reply:
trying out the new output structure before it's out there, and assessing what we want to change/update